src/services/entity-types.js

```diff
@@ -25,6 +25,31 @@ module.exports = class EntityTypeHelper {
 			bodyData.updated_by = loggedInUserId
 			bodyData.organization_id = orgId
 			bodyData.value = bodyData.value.toLowerCase()
+			bodyData.config = {}
+			if (bodyData?.is_external) {
+				bodyData.config = {
+					is_external: bodyData?.is_external ? true : false,
+				}
+			}
+
+			if (bodyData?.depended_on) {
+				const check_depended_entity_type = await entityTypeQueries.findOneEntityType({
```

Comment:
@adithyadinesh0412 for the variables we are using camelCase so please folow that, For the fields which using i db we can use snake case